script.js

```diff
@@ -0,0 +1,385 @@
+    arrayValues.forEach((el) => {
+      let x = document.querySelector(`.${el[0]}`);
+      if (el.length < 2) {
+        x.innerText = el[0];
+      } else if (el[0] === "enter") {
+      } else if (el[1] !== undefined) {
+        x.innerText = el[1];
+      }
+
+      if (
+        el[0] === "Backspace" ||
+        el[0] === "del" ||
+        el[0] === "enter" ||
+        el[0] === "rShift"
+      ) {
+        document.querySelector(".Backspace").after(clear);
+        document.querySelector(".rShift").after(clear);
+      }
+    });
+  }
+
+  if (isShift === true && localStorage.lang === "ENG") {
+    arrayValues.forEach((el) => {
+      let x = document.querySelector(`.${el[0]}`);
+      if (el.length < 2) {
+        x.innerText = el[0];
+      } else if (el[0] === "enter") {
+      } else if (el[2] !== undefined) {
+        x.innerText = el[2];
+      }
+    });
+  }
+
+  if (isShift === false && localStorage.lang === "RUS") {
+    arrayValues.forEach((el) => {
+      let x = document.querySelector(`.${el[0]}`);
+      if (el.length < 2) {
+        x.innerText = el[0];
+      } else if (el[0] === "enter") {
+      } else if (el[3] !== undefined) {
+        x.innerText = el[3];
+      }
+    });
+  }
+
+  if (isShift === true && localStorage.lang === "RUS") {
+    arrayValues.forEach((el) => {
+      let x = document.querySelector(`.${el[0]}`);
+      if (el.length < 2) {
+        x.innerText = el[0];
+      } else if (el[0] === "enter") {
+      } else if (el[4] !== undefined) {
+        x.innerText = el[4];
+      }
+    });
+  }
+}
+
+function addListenersOnKeys() {
+  keyboard.addEventListener("mousedown", function (event) {
+    textarea.focus();
+    if (event.target.classList[0] === "keyboard") return;
+    LastPress = event.target;
+    setStyle(event.target, true);
+    identifyKey(event.target.dataset, "down");
+    addKeysOnKeyboard();
+  });
+
+  keyboard.addEventListener("mouseup", function (event) {
+    textarea.focus();
+    if (event.target.dataset.code === "CapsLock") return;
+    removeStyle();
+    identifyKey(event.target.dataset, "up");
+    addKeysOnKeyboard();
+  });
+
+  document.addEventListener("keydown", function (event) {
+    if (event.repeat) return;
+    event.preventDefault();
+    setStyle(document.querySelector(`.${KEY_CODE[event.code][0]}`), true);
+    identifyKey(event, "down");
+    checkLang();
+    addKeysOnKeyboard();
+  });
+
+  document.addEventListener("keyup", function (event) {
+    if (event.repeat) return;
+    if (event.key === "CapsLock") return;
+    setStyle(document.querySelector(`.${KEY_CODE[event.code][0]}`), false);
+    identifyKey(event, "up");
+    checkLang();
+    addKeysOnKeyboard();
+  });
+}
+
+function setStyle(code, bool) {
+  if (bool) code.classList.add("active");
+  else code.classList.remove("active");
+}
+
+function removeStyle() {
+  LastPress.classList.remove("active");
+}
+
+function identifyKey(event, updown) {
+  if (event.code === "ShiftLeft") {
```

Comment:
maybe change this to `if (event.code === "ShiftLeft" && shiftCaseLeft){ shiftCaseLeft = false  } `